src/hound/vcs/svn.go

```diff
@@ -0,0 +1,79 @@
+package vcs
+
+import (
+	"bytes"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func init() {
+	RegisterVCS("svn", &SVNDriver{})
+}
+
+type SVNDriver struct{}
+
+func (g *SVNDriver) HeadHash(dir string) (string, error) {
+	cmd := exec.Command(
+		"svnversion")
+	cmd.Dir = dir
+	r, err := cmd.StdoutPipe()
+	if err != nil {
+		return "", err
+	}
+	defer r.Close()
+
+	if err := cmd.Start(); err != nil {
+		return "", err
+	}
+
+	var buf bytes.Buffer
+
+	if _, err := io.Copy(&buf, r); err != nil {
+		return "", err
+	}
+
+	return strings.TrimSpace(buf.String()), cmd.Wait()
+}
+
+func (g *SVNDriver) Pull(dir string) (string, error) {
+	cmd := exec.Command(
+		"svn",
+		"update",
+		"--username",
+		os.Getenv("SVN_USERNAME"),
```

Comment:
I only used the environment variables in this driver as the config struct for the repo was not passed into any of the drivers functions. I could change this though?